builtin-functions/_functions.txt

```diff
@@ -728,13 +728,14 @@ function setlocale ($category ::: int, $locale ::: string) ::: string | false;
 
 function iconv ($input_encoding ::: string, $output_encoding ::: string, $input_str ::: string) ::: string | false;
 
-function mb_check_encoding ($str ::: string, $encoding ::: string = "1251") ::: bool;
-function mb_strlen ($str ::: string, $encoding ::: string = "1251") ::: int;
-function mb_strpos ($haystack ::: string, $needle ::: string, $offset ::: int = 0, $encoding ::: string = "1251") ::: int | false;
-function mb_stripos ($haystack ::: string, $needle ::: string, $offset ::: int = 0, $encoding ::: string = "1251") ::: int | false;
-function mb_strtolower ($str ::: string, $encoding ::: string = "1251") ::: string;
-function mb_strtoupper ($str ::: string, $encoding ::: string = "1251") ::: string;
-function mb_substr ($str ::: string, $start ::: int, $length ::: mixed = PHP_INT_MAX, $encoding ::: string = "1251") ::: string;
+function mb_internal_encoding(?string $encoding = null): mixed;
+function mb_check_encoding ($str ::: string, $encoding ::: string = "") ::: bool;
+function mb_strlen ($str ::: string, $encoding ::: string = "") ::: int;
+function mb_strpos ($haystack ::: string, $needle ::: string, $offset ::: int = 0, $encoding ::: string = "") ::: int | false;
+function mb_stripos ($haystack ::: string, $needle ::: string, $offset ::: int = 0, $encoding ::: string = "") ::: int | false;
+function mb_strtolower ($str ::: string, $encoding ::: string = "") ::: string;
+function mb_strtoupper ($str ::: string, $encoding ::: string = "") ::: string;
+function mb_substr ($str ::: string, $start ::: int, $length ::: mixed = PHP_INT_MAX, $encoding ::: string = "") ::: string;
```

Comment:
I don't like changes in non-affected functions, it spoils blame :(

Reply:
Nevertheless it's required